README.rst

```diff
@@ -25,12 +25,12 @@ The ibmdbpy project translates Pandas-like syntax into SQL and uses a middleware
 
 The following scenario illustrates how ibmdbpy works.
 
-Assuming that all ODBC connection parameters are correctly set, issue the following statements to connect to a database (in this case, a Db2 instance named DASHDB) via ODBC:
+Assuming that all ODBC connection parameters are correctly set, issue the following statements to connect to a database (in this case, a Db2 database named BLUDB) via ODBC:
```

Comment:
I think it would be good to have @blhatdeibmcom take a look at this for all the doc changes, especially the `instance` -> `database` part.

Reply:
In this case it is obvious that instance is the wrong term, because DASHDB and BLUDB are names for databases and you cannot connect to an instance.